db/seeds.rb

```diff
@@ -41,5 +53,43 @@
 
 # Create Aircrafts
 aircrafts.each do |aircraft_data|
-  Aircraft.create!(aircraft_data)
+  aircraft = Aircraft.create!(aircraft_data)
+  AIRCRAFT_NOTIFICATIONS_COUNT.times do
+    notification = NewAircraftCreatedNotification.with(aircraft: aircraft)
+    notification.deliver_later(NewAircraftCreatedNotification.targets)
+  end
+end
+
+# Create Logbook Entries and Notifications
+LOGBOOK_ENTRY_NOTIFICATIONS.times do
+  logbook_entry = LogbookEntry.create!(
+    aircraft: Aircraft.order("RANDOM()").first,
+    pilot_in_command: users.sample,
+    second_in_command: [users.sample, nil].sample,
+    date: Faker::Date.backward(days: 365),
+    departure_icao: Faker::Address.country_code,
+    arrival_icao: Faker::Address.country_code,
+    duration: Faker::Number.between(from: 1, to: 10),
+    time_of_day: LogbookEntry.time_of_days.keys.sample
+  )
+  new_logbook_notification = NewLogbookEntryCreatedNotification.with(logbook_entry: logbook_entry)
+  new_logbook_notification.deliver_later(NewLogbookEntryCreatedNotification.targets)
+
+  update_logbook_notification = LogbookEntryUpdatedNotification.with(logbook_entry: logbook_entry)
+  update_logbook_notification.deliver_later(LogbookEntryUpdatedNotification.targets)
 end
+
+# Create 1000 UserSignUpNotification and UserUpdateNotification
+users.all.each do |user|
```

Comment:
```suggestion
users.each do |user|
```

not needed anymore since `users` will be an `Array`